File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/kotlin/valid.kt
// Fixture — café 日本語 before declarations
package fixture

import kotlin.math.abs

/** A greeter. */
class Greeter(val name: String) {
    /** Say hello. */
    fun hello(): String = "hi $name"
}

interface Shape {
    fun area(): Double
}

object Registry {
    fun lookup(id: Int): String = "x"
}

fun helper(x: Int): Int = abs(x)

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/scala/valid.scala
// Fixture — café 日本語 before declarations
package fixture

import scala.math.abs

/** A greeter. */
class Greeter(name: String) {
  /** Say hello. */
  def hello(): String = s"hi $name"
}

trait Shape {
  def area(): Double
}

object Registry {
  def lookup(id: Int): String = "x"
}

case class Pair(a: Int, b: Int)
